### PR TITLE
Replace theme with alabaster.

### PR DESCRIPTION
**Pull request by ashwineaso**
1. Replaced the documentation theme with alabaster

### CODE REVIEW

**Comment by ashwineaso** [comment]
I was thinking of updating the docs to look similar to https://2.python-requests.org/en/master/. Please advice on what to include in this PR.

**Comment by jadchaar** [comment]
The Alabaster theme is adapted from Kenneth Reitz's Requests theme, so it should be fine as is. Let me take a look at your PR.

File: docs/_themes/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*.pyc
```
Comment (jadchaar) [inline docs/_themes/.gitignore]: This is fine, but these ignores are already included in the base directory's `.gitignore` file.

File: docs/conf.py [hunk 2]
```diff
@@ -1,64 +1,250 @@
...
 
-html_theme = "f6"
-html_theme_path = ["_themes"]
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+#
+html_theme_options = {
+    "show_powered_by": False,
+    "github_user": "arrow",
```
Comment (jadchaar) [inline docs/conf.py]: This should probably be `crsmithdev`.

File: docs/index.rst [hunk 3]
```diff
@@ -1,3 +1,8 @@
+.. arrow documentation master file, created by
```
Comment (jadchaar) [inline docs/index.rst]: Let's remove this comment to keep this file clean :).

File: docs/conf.py (hunk 2 above)
Comment (ashwineaso) [reply docs/conf.py]: Yes. Sorry about that.

File: docs/index.rst (hunk 3 above)
Comment (ashwineaso) [reply docs/index.rst]: Done.

**Comment by ashwineaso** [comment]
Made all the changes you suggested. And cleaned up the mess, I hope. Please review once more.

**Comment by jadchaar** [comment]
Looks good to me. Thanks for the contribution @ashwineaso 🎉!

**Pull request merged**